rudder-core/src/main/scala/com/normation/rudder/repository/jdbc/ExpectedReportsJdbcRepository.scala

```diff
@@ -89,6 +90,14 @@ class PostgresqlInClause(
     //use IN ( VALUES (), (), ... )
     else s"${attribute} IN(VALUES ${values.mkString("('","'),('","')")})"
   }
+
+  def inNumber(attribute: String, values: Iterable[AnyVal]): String = {
+    //with values, we need more ()
+    if(values.isEmpty) ""
+    else if(values.size < inClauseMaxNbElt) s"${attribute} IN (${values.mkString("", ",", "")})"
```

Comment:
As explain in the comment above, it's an optimization with postgres interval behaviour.